src/chapters/orm-part1/repositories.rst

```diff
@@ -0,0 +1,51 @@
+Repositories
+============
+
+.. index:: ! @Autowired
+
+Now that we have a general idea of how to set up our entity classes and repositories, let's take a deeper dive into how repositories are used to perform CRUD operations on a database.
+
+CRUD Operations
+---------------
+
+Previously, we learned that we can create a repository to fetch rows of a table in a database.
+To do so, we need to create a corresponding controller. 
+When creating our controller, we add a variable of the repository type that has the annotation ``@Autowired``.
+
+Next, we will learn how to use some of the more common ``CrudRepository`` methods, such as ``delete(T entity)``, ``deleteById(ID id)``, ``findAll()``, ``findById(ID id)``, and ``save(S entity)``.
+
+Using Persistence in a Controller - Video
+-----------------------------------------
+
+The following video explains how we can add a controller to manage persistence in our ``coding-events`` application. 
+The accompanying text is a quick rundown of what happens in the video. To get started, create a branch off of your `persistent-model <https://github.com/LaunchCodeEducation/coding-events/tree/persistent-model>`_ branch.
+
+.. raw:: html
+
+   <div style="text-align:center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/0eug2HI7rbo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+
+Using Persistence in a Controller - Text
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a controller called ``EventController``. We will add request mapping to ``"events"``.
+Once we do that, we can add a variable of type ``EventRepository`` and add an ``@Autowired`` annotation.
+
+For events, we want to be able to display all of the events in our database and add and delete events.
```

Comment:
I thinks "as well as" works better here than "and"

Reply:
Done!